ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,308 @@
+                    } else {
+                        Text(section.title ?? "")
+                            .listRowInsets(EdgeInsets(
+                                    top: tokens.horizontalCellPadding / 2,
+                                    leading: tokens.horizontalCellPadding,
+                                    bottom: tokens.horizontalCellPadding / 2,
+                                    trailing: tokens.horizontalCellPadding))
+                            .font(Font(tokens.subtitleFont))
+                            .foregroundColor(Color(tokens.subtitleColor))
+                    }
+                }
+                ForEach(section.cells, id: \.self) { cell in
+                    MSFListCellView(cell: cell, layoutType: section.layoutType, tokens: tokens)
+                        .border(section.hasBorder ? Color(tokens.borderColor) : Color.clear, width: section.hasBorder ? tokens.borderSize : 0)
+                        .frame(maxWidth: .infinity)
+                }
+            }
+        }
+        .environment(\.defaultMinListRowHeight, 0)
+        .frame(maxWidth: .infinity, maxHeight: .infinity)
+    }
+}
+
+extension MSFListView {
+    /// View for List Cells
+    struct MSFListCellView: View {
+        var cell: MSFListVnextCellData
+        var layoutType: MSFListCellVnextLayoutType
+        var tokens: MSFListTokens
+
+        init(cell: MSFListVnextCellData, layoutType: MSFListCellVnextLayoutType, tokens: MSFListTokens) {
+            self.cell = cell
+            self.layoutType = layoutType
+            self.tokens = tokens
+        }
+
+        var body: some View {
+            Button(action: cell.onTapAction ?? {}, label: {
+                HStack(spacing: 0) {
+                    if let leadingIcon = cell.leadingIcon {
+                        Image(uiImage: leadingIcon)
+                            .resizable()
+                            .frame(width: tokens.iconSize, height: tokens.iconSize)
+                            .padding(.trailing, tokens.iconInterspace)
+                    }
+                    VStack(alignment: .leading) {
+                        if let title = cell.title {
+                            Text(title)
+                                .font(Font(tokens.textFont))
+                                .foregroundColor(Color(tokens.leadingTextColor))
+                                .lineLimit(cell.titleLineLimit)
+                        }
+                        if let subtitle = cell.subtitle, !subtitle.isEmpty {
+                                Text(subtitle)
+                                    .font(Font(tokens.subtitleFont))
+                                    .foregroundColor(Color(tokens.subtitleColor))
+                                    .lineLimit(cell.titleLineLimit)
+                        }
+                    }
+                    Spacer()
+                    HStack {
+                        if let trailingIcon = cell.trailingIcon {
```

Comment:
Instead of the 2 consecutive if statements:

```
                        if let accessoryType = cell.accessoryType,
                           accessoryType != .none,
                           let accessoryIcon = accessoryType.icon {
```

(provided you took my suggestion of renaming **trailingIcon** to **accessoryType** above)